api/soap/mantisconnect.wsdl

```diff
@@ -163,6 +163,7 @@
    <xsd:element name="monitors" type="tns:AccountDataArray" minOccurs="0"/>
    <xsd:element name="sticky" type="xsd:boolean" minOccurs="0"/>
    <xsd:element name="tags" type="tns:ObjectRefArray" minOccurs="0"/>
+   <xsd:element name="token" type="xsd:string" minOccurs="0"/>
```

Comment:
@vboctor 

Fogbugz returns just token (not full URL), so I just followed the same practice. I assume that is because one can access your server via other URL than saved in your settings? Not sure, but I think this is not a great deal.

It is read-only server-generated value - same as bug ID, so it is totally ignored for "add"/"update".